### ISSUE TITLE
Accessibility improvements on default color palette

### ISSUE THREAD

**Issue by muratcorlu** [Trendyol/baklava#222]
Our current default color palette has some accessibility issues because of insufficient contrast ratio. We always want to keep a11y on our mind while we develop Baklava Design System, so this issue is an important concern for us.

Our design team is already working on this, and this issue is for following the work on that topic. Updates and fixes can be tracked here.

**Comment by batuhantozun** [comment]
We'll fix on #91
